manifests/pxeboot/kickstart.ks

```diff
@@ -203,6 +203,52 @@ fi
 
 ################################################################################
 
+cat <<'EOF' > /usr/bin/run_octep_cp_agent
+#!/bin/bash
+
+CMD="${1:-run}"
+IMAGE="${IMAGE:-quay.io/sdaniele/marvell-tools:latest}"
```

Comment:
Do you want to own the quay repo for this tool?

Reply:
No, that's not necessary.

For development, it's necessary that we can override this (otherwise, I could not have tested this PR). I can push a test version to `quay.io/thaller/marvell-tools:latest`. I use the `build_container.sh` script from this PR.

At all places where it matters, I can instead use my repo. For example, CDA runs this container, and there I can set `CDA_MARVELL_TOOLS_IMAGE=...`.

Reply:
Ok sounds good, just thinking if you wanted to make custom changes whether it was inconvenient or not.